gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/AccountFactorsEndpointHandler.java

```diff
@@ -653,16 +665,22 @@ private void sendChallenge(FactorProvider factorProvider,
                             error -> handler.handle(Future.failedFuture(error))
```

Comment:
missing log?

Reply:
Please look at the explanation on the other place (below few comments) for the similar point you made.